<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppCafe Repository</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SoftwareManager.qrc">
    <normaloff>:/icons/appcafe.png</normaloff>:/icons/appcafe.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Note: It may take a couple minutes to apply changes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radio_production">
        <property name="text">
         <string>Production (Updates Quarterly)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_edge">
        <property name="text">
         <string>Edge (Updates Frequently)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_custom">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="group_custom">
        <property name="title">
         <string>Custom Repositories (only 1 can be active)</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QToolButton" name="tool_addRepo">
             <property name="toolTip">
              <string>Add new custom repository</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="SoftwareManager.qrc">
               <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="tool_remRepo">
             <property name="toolTip">
              <string>Remove currently selected repository</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="SoftwareManager.qrc">
               <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="push_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="SoftwareManager.qrc">
         <normaloff>:/icons/dialog-cancel.png</normaloff>:/icons/dialog-cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="push_apply">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="SoftwareManager.qrc">
         <normaloff>:/icons/dialog-ok.png</normaloff>:/icons/dialog-ok.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="SoftwareManager.qrc"/>
 </resources>
 <connections/>
</ui>
